pkg/storkctl/snapshot.go

```diff
@@ -220,3 +223,153 @@ func getSnapshotStatusAndTime(snap *snapv1.VolumeSnapshot) (string, string) {
 	}
 	return string(snapv1.VolumeSnapshotConditionPending), toTimeString(time.Time{})
 }
+
+func newCreateVolumeSnapshotRestoreCommand(cmdFactory Factory, ioStreams genericclioptions.IOStreams) *cobra.Command {
+	var snapName string
+	var snapNamespace string
+	var snapGroup bool
+
+	restoreSnapshotCommand := &cobra.Command{
+		Use:     restoreSubCommand,
+		Aliases: []string{"restore"},
+		Short:   "Restore snapshot to source PVC",
+		Run: func(c *cobra.Command, args []string) {
+			if len(args) != 1 {
+				util.CheckErr(fmt.Errorf("exactly one argument needs to be provided for volumesnapshotrestore name"))
+				return
+			}
+			restoreCRDName := args[0]
+			snapRestore := &storkv1.VolumeSnapshotRestore{
+				Spec: storkv1.VolumeSnapshotRestoreSpec{
+					SourceName:      snapName,
+					SourceNamespace: snapNamespace,
+					GroupSnapshot:   snapGroup,
+				},
+			}
+			snapRestore.Name = restoreCRDName
+			snapRestore.Namespace = cmdFactory.GetNamespace()
+			_, err := k8s.Instance().CreateVolumeSnapshotRestore(snapRestore)
+			if err != nil {
+				util.CheckErr(err)
+				return
+			}
+
+			msg := fmt.Sprintf("Snapshot %v restored successfully", restoreCRDName)
```

Comment:
This should be `VolumeSnapshot restore %v started successfully` since the actual restore can take some time